src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -155,6 +156,11 @@ protected static IOutput Output
         /// </summary>
         public Verbosity VerbosityLevel => verbosityLevel;
 
+        /// <summary>
+        /// Dictionary of summary of each source. 
```

Comment:
>Dictionary of summary of each source.  [](start = 12, length = 38)

nit: Source level summary of test result

Reply:
changed